data_runner/README.md

```diff
@@ -0,0 +1,41 @@
+# Twenty Forty Eight
+
+## Overview
+
+The data runner is trapped in an infinite library, searching for your metrics. Open up a portal to watch him and see if you can get a better score than your peers! An idle game for the Datadog dashboard.
+
+### What it does
+
+Add the Data Runner widget to your dashboard, and choose a metric you want him to find. He'll run around the infinite library looking for it, and every time he finds it, the metric's value is added to your score. Leave him to run and see what you're up to at the end of the day!
```

Comment:
I would rewrite it like this: "Add the Data Runner widget to your dashboard, then choose a metric you want him to find. He'll run around the infinite library looking for it, and every time he finds it the metric's value will be added to your score. Leave him to run and see what your score is at the end of the day!"